groovy/postInit/mod/GregTech.groovy

```diff
@@ -410,6 +410,15 @@ mods.gregtech.chemical_bath.recipeBuilder()
         .EUt(30)
         .buildAndRegister();
 
+//Iron III chloride
+
+mods.gregtech.fluid_solidifier.recipeBuilder()
```

Comment:
this recipe should already be autogenerated if a material has both a dust and a fluid material form

Reply:
its not